bearer/bearer.go

```diff
@@ -261,7 +262,7 @@ func (b *Token) ForUser(id user.ID) {
 //
 // See also ForUser.
 func (b Token) AssertUser(id user.ID) bool {
-	return !b.targetUserSet || b.targetUser == id
+	return b.targetUser.IsZero() || b.targetUser == id
```

Comment:
May be change `b.targetUser == id` -> `b.targetUser.Equals(id)`?

Reply:
`Equals` deprecated already

Reply:
Ok, I see